com.unity.render-pipelines.high-definition/Editor/Material/ShaderGraph/HDSubShaderUtilities.cs

```diff
@@ -233,7 +233,8 @@ public static System.Collections.Generic.List<HDRenderQueue.RenderQueueType> Get
             }
             else
             {
-                result.Add(HDRenderQueue.RenderQueueType.PreRefraction);
+                if (!hasRefraction)
```

Comment:
thinking about that, what is happening if we are on prerefraction rendering pass and we select a refraction mode . In shader code it will work (refraction is just disabled), but in the UI. cc @TomasKiniulis

Reply:
I guess it is the same problem than for blend mode, we will need to always display the value but have a warning to say that it is not working. Thought?

Reply:
When I tested it was automatically setting the rendering pass to Default instead of Pre-Refraction

Reply:
I added the warning in both ShaderGraph UI and material inspector.

As a side effect, the refraction UI is now always visible even if we select pre-refraction rendering pass

![image](https://user-images.githubusercontent.com/6877923/96600132-e9450e00-12f0-11eb-8ebe-398fe7029495.png)